packages/in_app_purchase/in_app_purchase_ios/ios/Classes/FIAPReceiptManager.m

```diff
@@ -9,18 +9,22 @@ @implementation FIAPReceiptManager
 
 - (NSString *)retrieveReceiptWithError:(FlutterError **)error {
   NSURL *receiptURL = [[NSBundle mainBundle] appStoreReceiptURL];
-  NSData *receipt = [self getReceiptData:receiptURL];
+  NSError *err;
+  NSData *receipt = [self getReceiptData:receiptURL error:&err];
+  if (err) {
+    *error = [FlutterError errorWithCode:[[NSString alloc] initWithFormat:@"%li", (long)err.code]
```

Comment:
ObjC convention is that error pointer pointer parameters can be null, so this should check before assigning.